FV3/atmos_cubed_sphere/model/dyn_core.F90

```diff
@@ -671,7 +683,10 @@ subroutine dyn_core(npx, npy, npz, ng, sphum, nq, bdt, n_split, zvir, cp, akap,
       !$ser savepoint PGradC-Out
       !$ser data uc=uc vc=vc
 
-                                                                   call timing_on('COMM_TOTAL')
+                                                                  call timing_on('COMM_TOTAL')
+                                                   
+      !$ser savepoint HaloVectorUpdate-In 
+      !$ser data complete=complete_halo_true array_u=uc array_v=vc rank=tile_id
```

Comment:
Is it not possible to just write `complete=.true.` here?

Reply:
It's not! I know, I was surprised as well.

Reply:
This is a nit, but can we call this something generic like `true_boolean` then? A variable name specific to completing halo updates makes me initially think "huh, what kind of type is this that would be specific to halo updates? Why doesn't `.true.` work?

Reply:
sure -- I can change that variable name.